components/CurrentLocation.js

```diff
@@ -1,9 +1,14 @@
-/* eslint-disable no-undef */
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import coord from "../constants/buildingCoordinates";
 import { isPointInPolygon } from "geolib";
-import { AsyncStorage } from "react-native";
+import { AsyncStorage, StyleSheet } from "react-native";
 import * as Location from "expo-location";
+import Modal from "react-native-modal";
+import Icon from "react-native-vector-icons/Entypo";
+
+import { View, Text, Button, TouchableOpacity } from "react-native";
```

Comment:
import react-native twice (line 4 and line 9)

Reply:
Got it !